parm/config/config.resources

```diff
@@ -202,6 +203,71 @@ elif [[ "${step}" = "atmanlfinal" ]]; then
     export npe_node_atmanlfinal
     export is_exclusive=True
 
+elif [[ "${step}" = "landanlinit" || "${step}" = "landanlfinal" ]]; then
+
+   # below lines are for creating JEDI YAML
+   case ${CASE} in
+     C768)
+        layout_x=6
+        layout_y=6
+        ;;
+     C384)
+        layout_x=5
+        layout_y=5
+        ;;
+     C192 | C96 | C48)
+        layout_x=1
+        layout_y=1
+        ;;
+     *)
+        echo "FATAL ERROR: Resolution not supported for land analysis'"
+        exit 1
+   esac
+
+   export layout_x
+   export layout_y
+
+   declare -x "wtime_${step}"="00:10:00"
+   declare -x "npe_${step}"=1
+   declare -x "nth_${step}"=1
+   temp_stepname="nth_${step}"
+   declare -x "npe_node_${step}"="$(echo "${npe_node_max} / ${!temp_stepname}" | bc)"
+   declare -x "memory_${step}"="3072M"
+
+elif [[ "${step}" = "landanlrun" ]]; then
+
+   case ${CASE} in
+     C768)
+        layout_x=6
+        layout_y=6
+        ;;
+     C384)
+        layout_x=5
+        layout_y=5
+        ;;
+     C192 | C96 | C48)
+        layout_x=1
+        layout_y=1
+        ;;
+     *)
+        echo "FATAL: Resolution not supported'"
+        exit 1
+   esac
+
+   export layout_x
+   export layout_y
+
+   export wtime_landanlrun="00:30:00"
+   npe_landanlrun=$(echo "${layout_x} * ${layout_y} * 6" | bc)
+   export npe_landanlrun
+   npe_landanlrun_gfs=$(echo "${layout_x} * ${layout_y} * 6" | bc)
+   export npe_landanlrun_gfs
+   export nth_landanlrun=1
+   export nth_landanlrun_gfs=1
+   npe_node_landanlrun=$(echo "${npe_node_max} / ${nth_landanlrun}" | bc)
+   export npe_node_landanlrun
+   export is_exclusive=True
```

Comment:
Since th settings in the JEDI YAML need to be consistent between the init, run and final, I am making this suggestion.  What do you think?
```suggestion
elif [[ "${step}" = "landanlinit" || "${step}" = "landanlrun"  || "${step}" = "landanlfinal" ]]; then

   # below lines are for creating JEDI YAML
   case ${CASE} in
     C768)
        layout_x=6
        layout_y=6
        ;;
     C384)
        layout_x=5
        layout_y=5
        ;;
     C192 | C96 | C48)
        layout_x=1
        layout_y=1
        ;;
     *)
        echo "FATAL ERROR: Resolution not supported for land analysis'"
        exit 1
   esac

   export layout_x
   export layout_y

   if [[ "${step}" = "landanlinit" || "${step}" = "landanlfinal" ]]; then 
       declare -x "wtime_${step}"="00:10:00"
       declare -x "npe_${step}"=1
       declare -x "nth_${step}"=1
       temp_stepname="nth_${step}"
       declare -x "npe_node_${step}"="$(echo "${npe_node_max} / ${!temp_stepname}" | bc)"
       declare -x "memory_${step}"="3072M"
   elif [[ "${step}" = "landanlrun" ]]; then
       export wtime_landanlrun="00:30:00"
       npe_landanlrun=$(echo "${layout_x} * ${layout_y} * 6" | bc)
       export npe_landanlrun
       npe_landanlrun_gfs=$(echo "${layout_x} * ${layout_y} * 6" | bc)
       export npe_landanlrun_gfs
       export nth_landanlrun=1
       export nth_landanlrun_gfs=1
       npe_node_landanlrun=$(echo "${npe_node_max} / ${nth_landanlrun}" | bc)
       export npe_node_landanlrun
       export is_exclusive=True
   fi
```

Also, is the `_gfs` required in the `landanlrun` section.  Even if it is, I can't see it is correct. For e.g. `npe_node_landanlrun_gfs` is not calculated.

Reply:
Yes I like @aerorahul 's suggestion. We, for now, should just assume resources are identical between GFS and GDAS analyses, and revisit that if needed for time limits. My guess is the land anl will run in 10% of the time the atm anl will...

Reply:
@aerorahul `npe_landanlrun_gfs` is added here following the approach from the `aeroanlrun` section. @CoryMartin-NOAA Can I remove these `npe_landanlrun_gfs` here? Thanks.

Reply:
> Since th settings in the JEDI YAML need to be consistent between the init, run and final, I am making this suggestion. What do you think?
> 
I made the corresponding changes as suggested. Thanks for your efforts.